docs/spec/spec-proposals/f1-fee-distribution/f1_fee_distr.tex

```diff
@@ -17,49 +17,69 @@
+
+Even distribution of a validators rewards amongst its validators weighted by stake means the following: 
+Suppose a delegator delegates $x$ stake to a validator $v$ at block $h$.
 Let the amount of stake the validator has at block $i$ be $s_i$ and the amount of fees they receive at this height be $f_i$.
 Then if a delegator contributing $x$ stake decides to withdraw at block $n$, the rewards they receive is 
 $$\sum_{i = h}^{n} \frac{x}{s_i}f_i = x \sum_{i = h}^{n} \frac{f_i}{s_i}$$
 
-However $s_i$ will not change every block.
-It only changes if the validator gets slashed, or if someone new has bonded or unbonded.
-Handling slashes is relegated to \autoref{ssec:slashing}.
-Define a period as the set of blocks between two changes in a given validator's total stake.
+Note that $s_i$ does not change every block,
+it only changes if the validator gets slashed, or if someone new has bonded or unbonded.
+We'll relegate handling of slashes to \autoref{ssec:slashing}, and only consider the case with no slashing here.
+We can change the iteration from being over every block, to instead being over the set of blocks between two changes in validator $v$'s total stake.
+Let each of these set of blocks be called a period.
 A new period begins every time that validator's total stake changes.
-The above iteration will be converted to iteration over periods.
 Let the total amount of stake for the validator in period $p$ be $n_p$.
-Let $T_p$ be the total fees this validator accrued within this period.
+Let $T_p$ be the total fees that validator $v$ accrued in period $p$.
 Let $h$ be the start of period $p_{init}$, and height $n$ be the end of $p_{final}$.
 It follows that 
 $$x \sum_{i = h}^{n} \frac{f_i}{s_i} = x \sum_{p = p_{init}}^{p_{final}} \frac{T_p}{n_p}$$
 
-Let $p_0$ represent the period from when the validator first bonded until the first change to the validators stake.
-The central idea to the F1 model is that at the end of the $k$th period, the following is stored at a state location indexable by $k$: $\sum_{i=0}^{k}\frac{T_i}{n_i}$.
-When a delegator wants to delegate or withdraw their reward, they first create a new entry in state to end the current period. Let the index of the current period be $f$. 
-Then this entry is created using the previous entry as follows: $$\sum_{i=0}^{f}\frac{T_i}{n_i} = \sum_{i=0}^{f-1}\frac{T_i}{n_i} + \frac{T_f}{n_f} = entry_{f-1} + \frac{T_f}{n_f}$$
+Let $p_0$ represent the period which begins when the validator first bonds.
+The central idea to the F1 model is that at the end of the $k$th period,
+the following is stored at a state location indexable by $k$: $\sum_{i=0}^{k}\frac{T_i}{n_i}$.
+Let the index of the current period be $f$. 
+When a delegator wants to delegate or withdraw their reward, they first create a new entry in state to end the current period.
```

Comment:
Or redelegate, or unbond (any stake change), right?

Reply:
Yeah, I think the initial exposition is simplified by only considering delegation and withdrawals, and the rest can be described in a later section.